src/pages/profile-create.page.tsx

```diff
@@ -0,0 +1,180 @@
+              orientation="horizontal"
+              sx={{ width: "100%", justifyContent: "center", flexDirection: { xs: "column", md: "row" }, p: 0, gap: 2 }}
+            >
+              {USER_ROLES.map((role, index) => (
+                <ListItem key={index}>
+                  <ListItemButton
+                    onClick={() => setProfile(prev => ({ ...prev, selectedRole: role }))}
+                    sx={{
+                      bgcolor:
+                        profile.selectedRole === role
+                          ? getColorTransparency(getColorHex("primary"), 10)
+                          : "transparent",
+                      width: 160,
+                      justifyContent: "center",
+                      border: "1px solid",
+                      borderColor: profile.selectedRole === role ? "subvisual.primary" : "neutral.lighter",
+                      borderRadius: 8
+                    }}
+                  >
+                    {role}
+                  </ListItemButton>
+                </ListItem>
+              ))}
+            </List>
+          )}
+
+          {activeStep === 1 && (
+            <Stack width="100%" gap={4}>
+              <UserAvatar avatar={profile.selectedAvatar} />
+
+              <Stack bgcolor="neutral.white" overflow="auto">
+                <List orientation="horizontal" sx={{ p: 2 }}>
+                  {memojis.map((emojiSrc, index) => (
+                    <ListItem key={index}>
+                      <ListItemButton
+                        onClick={() => setProfile(prev => ({ ...prev, selectedAvatar: emojiSrc }))}
+                        sx={{
+                          bgcolor: "neutral.lightest",
+                          width: 120,
+                          border: "2px solid",
+                          borderColor: profile.selectedAvatar === emojiSrc ? "subvisual.primary" : "neutral.lightest",
+                          borderRadius: 8,
+                          mr: 0.5
+                        }}
+                      >
+                        <img src={emojiSrc} alt={`Memoji ${index + 1}`} style={{ height: "auto", width: "100%" }} />
+                      </ListItemButton>
+                    </ListItem>
+                  ))}
+                </List>
+              </Stack>
+            </Stack>
+          )}
+
+          <Stack direction="row" justifyContent={activeStep === 0 ? "end" : "space-between"} alignItems="center">
+            {activeStep === 1 && (
+              <JoyLink
+                onClick={() => setActiveStep(0)}
+                underline="none"
+                startDecorator={<ArrowRightOutlined sx={{ fontSize: 10, transform: "rotate(180deg)" }} />}
+                textColor="neutral.black"
+                gap={0.5}
+              >
+                Back
+              </JoyLink>
+            )}
+            <JoyLink
+              onClick={() => {
+                if (activeStep === 0) {
+                  if (!profile.selectedRole) {
```

Comment:
You could also just make JoyLink disabled if !profile.selectedRole, preventing this case